mode/examples/Topics/Image Processing/Blending/Blending.pyde

```diff
@@ -21,17 +18,17 @@ modes = ((REPLACE, "REPLACE"),
          (MULTIPLY, "MULTIPLY"),
          (SCREEN, "SCREEN"),
          (REPLACE, "REPLACE"))
+img1 = loadImage("layer1.jpg")
```

Comment:
Well, for what it's worth, I do prefer this new way; I just wanted to be sure that it does work.